### DISCUSSION TITLE
Unable to Access GitHub Copilot

### DISCUSSION THREAD

**Discussion by boules131** [community/community#116913] [GitHub Education]
### Hi everyone,

![Screenshot 2024-04-05 141141](https://github.com/community/community/assets/135156301/1596e7b4-bc8c-4049-bf6e-6786a0d63431)
![Screenshot 2024-04-05 141617](https://github.com/community/community/assets/135156301/accfcc50-1c8f-4737-b0fc-79165750812e)
I did all this and nothing changed

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
